.grype.yaml

```diff
@@ -60,3 +68,11 @@ ignore:
     package:
       type: npm
       name: qs
+  - vulnerability: GHSA-p8p7-x288-28g6
+    package:
+      type: npm
+      name: request
+  - vulnerability: GHSA-rc47-6667-2j5j
```

Comment:
Does this require a new entry in POLICY_CONFIGURATION.md ?

Reply:
Yes, missed these two bottom ones, will update